src/centroiders.cpp

```diff
@@ -76,17 +76,15 @@ int DetermineCutoff(unsigned char *image, int imageWidth, int imageHeight) {
 
 std::vector<Star> CenterOfGravityAlgorithm::Go(unsigned char *image, int imageWidth, int imageHeight) const {
     CentroidParams p;
-    std::unordered_set<int> checkedIndices;
+    //std::unordered_set<int> checkedIndices;
```

Comment:
Now that you've figured out this bug, let's remove the comments

Reply:
done